graphql-dgs-client/src/test/kotlin/com/netflix/graphql/dgs/client/GraphQLResponseTest.kt

```diff
@@ -238,4 +240,21 @@ class GraphQLResponseTest {
 
         server.verify()
     }
+
+    @Test
```

Comment:
We should probably add a test case for e.g. `response.extractValue("datasomething")`